typings/index.d.ts

```diff
@@ -3,7 +3,8 @@
 declare module 'draft-js-export-html' {
     import draftjs = require("draft-js");
 
-    type BlockStyleFn = (block: draftjs.ContentBlock) => any;
+    type BlockStyleFn = (block: draftjs.ContentBlock) => RenderConfig;
+    type EntityStyleFn = (entity: draftjs.EntityInstance) => RenderConfig;
```

Comment:
From my understanding the optional/maybe operator `?` is only used for properties on interfaces/classes and parameters in functions. Values are always nullable in TypeScript and the `?` operator is used where the value can be skipped all together, ie `undefined`